rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/profile/DeviceState.java

```diff
@@ -148,7 +149,7 @@ public void process(TbContext ctx, TbMsg msg) throws ExecutionException, Interru
             latestValues = fetchLatestValues(ctx, deviceId);
         }
         boolean stateChanged = false;
-        if (msg.isTypeOf(POST_TELEMETRY_REQUEST)) {
+        if (msg.isTypeOf(POST_TELEMETRY_REQUEST) || msg.isTypeOf(TIMESERIES_UPDATED)) {
             stateChanged = processTelemetry(ctx, msg);
```

Comment:
Is it okay that you use the same method? E.g. for attributes there are two separate: processAttributesUpdate**Request** and processAttributesUpdate**Notification**